_posts/en/newsletters/2024-03-27-newsletter.md

```diff
@@ -7,14 +7,20 @@ type: newsletter
 layout: newsletter
 lang: en
 ---
-This week's newsletter FIXME
+This week's newsletter announces the disclosure of a bandwidth-wasting
+attack affecting Bitcoin Core and related nodes, describes several
+improvements to the idea for transaction fee sponsorship, and summarizes
+a discussion about using live mempool data to improve Bitcoin Core's
+feerate estimation.  Also included are our regular sections with
+selected questions and answers from the Bitcoin Stack Exchange,
+announcements of new releases and release candidates, and notable
+changes to popular Bitcoin infrastructure projects.
 
 ## News
 
-- **Disclosure of free relay attack:** Peter Todd [posted][todd free
-  relay] to the Bitcoin-Dev mailing list a bandwidth-wasting attack he
-  had previously [responsibly disclosed][topic responsible disclosures]
-  to the Bitcoin Core developers.  In short, Mallory broadcasts one
+- **Disclosure of free relay attack:** a bandwidth-wasting attack was
+  [described][topic responsible disclosures] to the Bitcoin-Dev mailing
```

Comment:
Shouldn't it be the link below?
https://gnusha.org/pi/bitcoindev/Zfg%2F6IZyA%2FiInyMx@petertodd.org/

Reply:
Thanks!!!!